db/migrate/20181011162644_create_matches.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMatches < ActiveRecord::Migration[5.2]
+  def change
+    create_table :matches do |t|
+      t.bigint :first_user_id, index: true, foreign_key: true
+      t.bigint :second_user_id, index: true, foreign_key: true
```

Comment:
Use ```t.references``` , It makes the code more readable.
Also make these columns not nullable. It's a good practice to add validations in model and database.